### DISCUSSION TITLE
Unable to connect to VEDO

### DISCUSSION THREAD

**Discussion by ward0** [gicamm/homeassistant-comelit#101] [General]
Configuration.yaml:
vedo:
host: 192.168.1.30
port: 80 (also tried other port 10012)
password: ******
scan_interval: 30

VEDO android app settings:

IP-Adres (lokaal IP-adres van de central): 192.168.1.30
Poort: 10012

Parameters remote modus:
IP-adres/Hostname: xxx.comelitdns.com
Poort centrale: 10012
Web-poort central: 2444
Poort Grab: 30012
https: enabled

I don't see any port 80 in my settings.

Router Telenet settings:
![image](https://github.com/user-attachments/assets/73526be5-5765-4d03-b677-f6a5be78c918)

**Comment by ward0** [comment]
I scanned my local network, no 192.168.1.230 found.
I tried all the ports on the 192.168.1.30 and http/ https combination, but I don't get any Vedo login page :(

**Comment by ward0** [comment]
http://192.168.1.30:6082 finally worked in my browser.

**Comment by ward0** [comment]
Next issue:
Why am I getting this?
![image](https://github.com/user-attachments/assets/3cc613a8-36b8-4e7c-b83e-58c4da00a400)

I can't control the alarm in any way.

How do I configure HA to fully use this integration? I activated the debug log in the HACS integration but no errors.

**Comment by ward0** [comment]
Fixed, in Entities I had to fill in the default password.
Is this normal / secure?
![image](https://github.com/user-attachments/assets/a985843f-2af4-456e-91b1-c6dcc78c4fe6)

**Reply by gicamm** [reply]
hi, the integration requires the password, otherwise cannot manage the VEDO.

**Reply by ward0** [reply]
> hi, the integration requires the password, otherwise cannot manage the VEDO.

Thx for the reply, hoping you will the 'Night' problem.
